gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 optfeature
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chm exif gsf pdf samba taglib test +unique"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/yelp-tools
+	dev-libs/atk
+	dev-libs/glib:2
+	gnome-base/gconf:2
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/harfbuzz:=
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/pango
```

Comment:
I see calls to `pango_` functions, but no `#include`s of pango headers nor checks in configure.ac for pango.

Reply:
I see pango being linked to final libraries and binaries.

X86_64;/usr/lib64/gnome-commander/plugins/libfileroller.so;libfileroller.so;;libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,**libpangocairo-1.0.so.0**,libatk-1.0.so.0,libcairo.so.2,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,**libpangoft2-1.0.so.0**,**libpango-1.0.so.0**,libgobject-2.0.so.0,libglib-2.0.so.0,libharfbuzz.so.0,libfontconfig.so.1,libfreetype.so.6,libstdc++.so.6,libc.so.6,libgcc_s.so.1;x86_64
X86_64;/usr/bin/gnome-commander;;/usr/lib64/gnome-commander;libgcmd.so.0,libgmodule-2.0.so.0,libunique-1.0.so.0,libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,**libpango-1.0.so.0**,libexiv2.so.27,libgsf-1.so.114,libgobject-2.0.so.0,libglib-2.0.so.0,libstdc++.so.6,libm.so.6,libgcc_s.so.1,libc.so.6;x86_64